src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericUploadFile.js

```diff
@@ -62,7 +65,8 @@ export const GenericUploadFile = ({
       labels={labels}
       tooltipText={t(TranslationUtils.getParameterTooltipTranslationKey(parameterData.id), '')}
       acceptedFileTypes={defaultFileTypeFilter}
-      handleUploadFile={(event) => FileManagementUtils.prepareToUpload(event, updateParameterValue)}
+      shouldHideFileName={renameFileOnUpload}
```

Comment:
Why the function is called _shouldHideFilename_ instead of _shouldRenameFilename_ ?

Reply:
That's because the react component `UploadFile` in the _ui_ repo does not rename anything, it simply "hides" the filenames and replace them by a generic placeholder (e.g. "CSV file"), the prop should not mention any "renaming" in the generic component.